frontend/server/src/main/java/com/amazonaws/ml/mms/metrics/Metric.java

```diff
@@ -53,6 +55,12 @@ public Metric(String metricName, String value, String unit, Dimension... dimensi
         this.value = value;
         this.unit = unit;
         this.dimensions = Arrays.asList(dimensions);
+        try {
+            InetAddress ip = InetAddress.getLocalHost();
```

Comment:
This might be very slow in certain case. Better cache it somewhere.

Reply:
Moved this to config manager.